lib/phobos_db_checkpoint/cli.rb

```diff
@@ -53,23 +53,39 @@ def copy_migrations
         end
       end
 
+      desc 'migration NAME', 'Generates a new migration with the given name. Use dashes (-) as a separator, ex: add-new-column'
+      option :destination,
+             aliases: ['-d'],
+             default: 'db/migrate',
+             banner: 'Destination folder relative to your project'
+      def migration(name)
+        @new_migration_class_name = name.split('-').map(&:capitalize).join('')
+        file_name = "#{migration_number}_#{name.split('-').join('_')}.rb"
```

Comment:
For me is a bit weird to receive the name split by '-', but then nothing is stored with dashes.

I vote for receiving the name split by '_', and we reject non word chars `.gsub(/[^\w]*/, '')` http://rubular.com/r/ZKCakqZoZ2